clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -0,0 +1,112 @@
+// RUN: %clang_cc1 -x c++ -fsycl-is-device -std=c++11 -fsyntax-only -verify -pedantic %s
+
+// Test for Intel FPGA loop attributes applied not to a loop
+void foo() {
+  // expected-error@+1 {{intelfpga loop attributes must be applied to for, while and do statements}}
```

Comment:
Maybe it's better to say: for, while OR do statements?

Reply:
Done